Makefile

```diff
@@ -38,10 +38,17 @@ CPPFLAGS = -Wall -I$(SRC)include -std=$(STD) -pthread \
 ARFLAGS = rcs
 LDFLAGS = -pthread
 LDLIBS = $(TEST_LIBS)
+LN = ln
+CP = cp
+MKDIR = mkdir
 
 SUDO =
 
 CCTZ_LIB = libcctz.a
+CCTZ_STATIC_LIB = $(CCTZ_LIB)
+CCTZ_SHARED_LIB = libcctz.so.2.0
```

Comment:
Is "2.0" referring to "CCTZ Version 2"?  If so, perhaps we should be providing some more central versioning information.